### PR TITLE
Fixe reformatting and refactor colab_utils.py

### PR DESCRIPTION
**Pull request by ambaha1**
- Add missing space after previous code reformatting
- Merge duplicate functions of the progress bar display
- Merge duplicate functions of pexpect (write and run the shell commands train_eval and inference)

### CODE REVIEW

File: src/colab/colab_utils.py [hunk 1]
```diff
@@ -65,38 +65,12 @@ def create_bucket(project, location, bucket_name):
...
-          <progress
-              value='{value}'
-              max='{max}'
-          >
-            {value}%
-          </progress>
-        """.format(
-      value=value, max=max)
-  return HTML(css + html)
+def get_id_project(project):
```
Comment (jzxu) [inline src/colab/colab_utils.py:68]: Can you rename to "get_project_id"?

File: src/colab/colab_utils.py [hunk 2]
```diff
@@ -212,18 +185,39 @@ class ProgressBar:
...
+        self.get_html(metrics, display_message), display_id=True)
 
-  def get_html(self, value, max):
-    return HTML(
-        f'Num generated examples: {value}/{max}'
-        f'<progress value="{value}" max="{max}" '
-        f'style="width: 100%">{value}</progress>'
-    )
+  def format_example_metrics(self, args):
+    return '''Num generated examples: {value}/{max}'''.format(**args)
```
Comment (jzxu) [inline src/colab/colab_utils.py:194]: Can just use ' instead of '''

File: src/colab/colab_utils.py [hunk 3]
```diff
@@ -924,6 +894,16 @@ def _download_eval_job_log(eval_job_id):
     return None
 
 
+def get_id_train_eval_job(project, location, job_name):
```
Comment (jzxu) [inline src/colab/colab_utils.py:897]: Can you rename to "get_train_eval_job_id"?

**Review by jzxu — CHANGES_REQUESTED** [review]
Thanks for creating this PR!

File: src/colab/colab_utils.py (hunk 1 above)
Comment (ambaha1) [reply src/colab/colab_utils.py]: done

File: src/colab/colab_utils.py (hunk 2 above)
Comment (ambaha1) [reply src/colab/colab_utils.py]: done

File: src/colab/colab_utils.py (hunk 3 above)
Comment (ambaha1) [reply src/colab/colab_utils.py]: done

**Review by jzxu — APPROVED** [review]
Thanks for addressing my comments!

**Pull request merged**